build/assets/master/0200_clusterrole.yaml

```diff
@@ -8,8 +8,8 @@ rules:
   resources:
   - pods
   - nodes
+  - nodes/status
```

Comment:
This is out of scope from the PR description, could you separate this commit into a new PR? thanks

Reply:
hi @ArangoGutierrez , thanks for the review, the rbac commit is deleted, and new PR for it was raised https://github.com/kubernetes-sigs/node-feature-discovery-operator/pull/79, please also have a look at it, thanks!